plugins/github-deployments/src/components/GithubDeploymentsTable/GithubDeploymentsTable.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { Table, TableColumn } from '@backstage/core';
+import { GithubDeployment } from '../../api';
+import moment from 'moment';
+import { Box, Typography, Link, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  empty: {
+    padding: theme.spacing(2),
+    display: 'flex',
+    justifyContent: 'center',
+  },
+}));
+
+const lastUpdated = (start: string): string => moment(start).fromNow();
+
+const State = ({ value }: { value: string }) => {
+  const colorMap: Record<string, string> = {
+    PENDING: 'orange',
+    IN_PROGRESS: 'orange',
+    ACTIVE: 'green',
+  };
+
+  return (
+    <Box display="flex" alignItems="center">
+      <span
+        style={{
+          display: 'block',
+          width: '8px',
+          height: '8px',
+          borderRadius: '50%',
+          backgroundColor: colorMap[value] || 'grey',
+          marginRight: '5px',
+        }}
+      />
+      <Typography variant="caption">{value}</Typography>
+    </Box>
+  );
+};
+
+const columns: TableColumn[] = [
+  {
+    title: 'Environment',
+    field: 'environment',
+    highlight: true,
+  },
+  {
+    title: 'Status',
+    render: (row: any): React.ReactNode => <State value={row.state} />,
```

Comment:
Can we type this row instead of `any`?

Reply:
I don't think so because of the type on render: `(property) Column<{}>.render?: ((data: {}, type: "row" | "group") => any) | undefined`

Reply:
I think if you do `TableColumn<{ state: string}>[]` does it clear up the types and you can remove the any?

Reply:
Here: https://github.com/backstage/backstage/pull/5114/files#diff-85ac04c660eadef50c087c0d75ba979a2f7411558ae3a1146664f0c23c2f5b41R56

Reply:
Ah, nice. That works, I've updated it now.